.github/actions/hugo-build-versioned-action/action.yaml

```diff
@@ -12,17 +12,28 @@ inputs:
     required: false
     description: Environment variable
     default: production
+  keep_master:
+    required: false
+    description: Preview of the master branch
+    default: no
+  fetch-from:
+    required: false
+    description: repo to fetch the versions from
+    default: origin
 
 runs:
   using: "composite"
   steps:
     - name: Generate Versioned Documentation
-      shell: bash 
+      shell: bash
       run: |
         cd docs
-        ../scripts/generate.sh origin master
+        rm generate.sh
+        wget https://raw.githubusercontent.com/gooddata/gooddata-python-sdk/master/scripts/generate.sh
+        chmod +x ./generate.sh
+        ./generate.sh ${{ inputs.fetch-from }} master ${{ inputs.keep_master}}
     - name: Hugo Build
-      uses: ./.github/actions/hugo-build-action
+      uses: gooddata/gooddata-python-sdk/.github/actions/hugo-build-action@master
```

Comment:
Actions were called from the branch, that was started the PR, could've been injected